tasks/sign-base64-blob/README.md

```diff
@@ -28,3 +28,5 @@ data:
 
 ## Changes in 1.0.1
 - Now the task decodes the payload from base64 before creating the `.sig` file
+## Changes in 1.0.2
```

Comment:
We typically put the newest to the top. Can you move this above 1.0.1 and add an empty line in between?